src/utils/customElements.ts

```diff
@@ -44,7 +47,16 @@ export function isElementAugmentable(element: HTMLElement | null): boolean {
  */
 export function shouldElementHaveHighlight(element: HTMLElement | null): boolean {
   const highlightType = getHighlightTypeForElement(element);
-  return highlightType !== HighlightType.None;
+
+  switch (highlightType) {
+    case HighlightType.None:
+      return false;
+    case HighlightType.Element:
+      return true;
+    default:
```

Comment:
nitpick: wouldn't it be better to mention all other highlight types here, so that it is more clear from code what `default` means. What do you think?

```
case HighlightType.ContentItem:
case HighlightType.ContentComponent:
default:
  return isInsideIFrame();
```

Reply:
My original idea was that a default branch implies that _every other_ highlight type than `ContentComponent` should be visible only in an iframe, while explicitly naming them suggests that only these specific cases should be handled differently - however, you are correct that this way it is more difficult to immediately understand what cases are covered. I've changed it to your proposed solution 😄